### PR TITLE
small nitpicks

### PR DESCRIPTION
**Pull request by basantagoswami**

### CODE REVIEW

File: 53.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Event `kind:1311` is live chat's channel message. Clients MUST include the `a` t
 
 ## Use Cases
 
-Common use cases include meeting rooms/workshops, watch-together activities, or event spaces, such as [live.snort.social](https://live.snort.social) and [nostrnests.com](https://nostrnests.com).
+Common use cases include meeting rooms/workshops, watch-together activities, or event spaces, such as [zap.stream](https://zap.stream).
```
Comment (AsaiToshiya) [inline 53.md:80]: Why nostrnests.com was removed? Is it dead?

Comment (basantagoswami) [reply 53.md:80]: Uses a different event kind. They will probably write a NIP or modify the live events one to add it. Or I will in a different PR.

**Comment by AsaiToshiya** [comment]
@basantagoswami Do you plan to make any other changes to this PR? If not, we have to undraft it and make it ready for merging.

**Comment by basantagoswami** [comment]
> Do you plan to make any other changes to this PR? If not, we have to undraft it and make it ready for merging.

I will make a bunch of other changes, yes. Just didn't get time after the initial draft to complete it.

**Review by AsaiToshiya — APPROVED** [review]
Let's merge this PR for now.

**Pull request merged**